### ISSUE TITLE
Maven crashes upon opening of certain libraries

### ISSUE THREAD

**Issue by Calicotobi** [eugenemel/maven#184]
```
Last login: Tue Mar 10 15:52:41 on ttys000
macbook-pro-4:~ tobiasmaile$ /Applications/Maven8-1-24-3.app/Contents/MacOS/Maven ; exit;
APP= "Maven2" VER= ""
Searching for method folder:
" Checking dir: /Applications/Maven8-1-22.app/Contents/MacOS/../Resources/methods"
METHODS FOLDER= "/Applications/Maven8-1-22.app/Contents/MacOS/../Resources/methods"
WRITE FOLDER= "/Users/tobiasmaile/Library/Application Support/Maven2"
Connecting to  "/Users/tobiasmaile/Library/Application Support/Maven2/ligand.db" 

Database::saveCompoundsSQL(): Saving 0 Compounds
Database::loadCompoundsSQL(): Elapsed time after count query: 0 msec
Database::loadCompoundSQL() Finished reading in data for "KNOWNS"
Database::loadCompoundSQL(): Finished appending data for "KNOWNS" to compoundDB in 0 msec. size= 0
loadAdducts() /Applications/Maven8-1-22.app/Contents/MacOS/../Resources/methods/ADDUCTS.csv count=42
Read in classification model /Applications/Maven8-1-22.app/Contents/MacOS/../Resources/methods/default.model
QObject::connect: No such signal BackgroundPeakUpdate::newPeakGroup(PeakGroup*,bool)
QObject::connect:  (receiver name: 'CalibrateDialog')
 EicWidget::recompute()
 EicWidget::computeEICs()
 EicWidget::replot(PeakGroup* group ) group= 0x0
 EicWidget::setFocusLine(float rt)
updateDatabaseList()
Database::loadCompoundsSQL(): Elapsed time after count query: 0 msec
Database::loadCompoundSQL() Finished reading in data for "KNOWNS"
Database::loadCompoundSQL(): Finished appending data for "KNOWNS" to compoundDB in 0 msec. size= 0
ligandwidget::showTable() 20170607-dipeptides-neg
setIonizationMode:  0
MassCalcWidget::compute()
setIonizationMode:  1
MassCalcWidget::compute()
changeUserAdduct::[M+H]+
 EicWidget::replot(PeakGroup* group ) group= 0x0
 EicWidget::setFocusLine(float rt)
GalleryWidget::resize() 
GalleryWidget::replot() 
updateDatabaseList()
objc[43693]: Class FIFinderSyncExtensionHost is implemented in both /System/Library/PrivateFrameworks/FinderKit.framework/Versions/A/FinderKit (0x7fff8cbb03d8) and /System/Library/PrivateFrameworks/FileProvider.framework/OverrideBundles/FinderSyncCollaborationFileProviderOverride.bundle/Contents/MacOS/FinderSyncCollaborationFileProviderOverride (0x110e61f50). One of the two will be used. Which one is undefined.
Load Library "MoNA-export-LC-MS_-_MSMS_-_Positive"
Database::loadCompoundsSQL(): Elapsed time after count query: 45 msec
Database::loadCompoundSQL() Finished reading in data for "MoNA-export-LC-MS_-_MSMS_-_Positive"
Segmentation fault: 11
logout
Saving session...
...copying shared history...
...saving history...truncating history files...
...completed.

[Process completed]
```

**Comment by PMSeitzer** [comment]
Also observed by @ngoccalico

**Comment by PMSeitzer** [comment]
To reproduce:

Open mzrollDB file in github issues 184 folder

**Comment by ngoccalico** [comment]
```
Last login: Tue Mar 10 15:50:32 on ttys002

The default interactive shell is now zsh.
To update your account to use zsh, please run `chsh -s /bin/zsh`.
For more details, please visit https://support.apple.com/kb/HT208050.
ngoc-mbp-1067:~ ngoc$ /Applications/Maven.app/Contents/MacOS/Maven ; exit;
APP= "Maven2" VER= ""
Searching for method folder:
" Checking dir: /Users/ngoc/Desktop/Maven.app/Contents/MacOS/../Resources/methods"
METHODS FOLDER= "/Users/ngoc/Desktop/Maven.app/Contents/MacOS/../Resources/methods"
WRITE FOLDER= "/Users/ngoc/Library/Application Support/Maven2"
Connecting to  "/Users/ngoc/Library/Application Support/Maven2/ligand.db" 

Database::saveCompoundsSQL(): Saving 0 Compounds
Database::loadCompoundsSQL(): Elapsed time after count query: 0 msec
Database::loadCompoundSQL() Finished reading in data for "KNOWNS"
Database::loadCompoundSQL(): Finished appending data for "KNOWNS" to compoundDB in 0 msec. size= 0
loadAdducts() /Users/ngoc/Desktop/Maven.app/Contents/MacOS/../Resources/methods/ADDUCTS.csv count=42
Database::loadPeakGroupTags(): 6 tags.
Read in classification model /Users/ngoc/Desktop/Maven.app/Contents/MacOS/../Resources/methods/default.model
QObject::connect: No such signal BackgroundPeakUpdate::newPeakGroup(PeakGroup*,bool)
QObject::connect:  (receiver name: 'CalibrateDialog')
 EicWidget::recompute()
 EicWidget::computeEICs()
 EicWidget::replot(PeakGroup* group ) group= 0x0
 EicWidget::setFocusLine(float rt)
updateDatabaseList()
Database::loadCompoundsSQL(): Elapsed time after count query: 0 msec
Database::loadCompoundSQL() Finished reading in data for "KNOWNS"
Database::loadCompoundSQL(): Finished appending data for "KNOWNS" to compoundDB in 0 msec. size= 0
ligandwidget::showTable() 20190926-Calico-Lipids-neg-PC-PE-oxygenated-diagnostic
setIonizationMode:  0
MassCalcWidget::compute()
setIonizationMode:  1
MassCalcWidget::compute()
updateDatabaseList()
 EicWidget::replot(PeakGroup* group ) group= 0x0
 EicWidget::setFocusLine(float rt)
 EicWidget::replot(PeakGroup* group ) group= 0x0
 EicWidget::setFocusLine(float rt)
rumsDB spectral library:  ""
opening..  "/Users/ngoc/Documents/MS_calico/X0136/X0136-M004A-GPX4-lipids/X0136-M004A-GPX4-lipids_mzkitchen_v2_11_output_Fri_Feb__7_11-04-49_PST_2020/X0136-M004A-GPX4-lipids.mzrollDB"
Database::loadCompoundsSQL(): Elapsed time after count query: 31 msec
Database::loadCompoundSQL() Finished reading in data for "ALL"
Database::loadCompoundSQL(): Finished appending data for "ALL" to compoundDB in 70 msec. size= 1767
Loading sample: "/Users/ngoc/Documents/MS_calico/X0136/X0136-M004A-GPX4-lipids/X0136-M004A-1_blank.mzML"
Found Blank: /Users/ngoc/Documents/MS_calico/X0136/X0136-M004A-GPX4-lipids/X0136-M004A-1_blank.mzML
loadSample time (msec) =  686
enumerateSRMScans: 11252
enumerateSRMScans: 11252
setIonizationMode:  -1
Loading sample: "/Users/ngoc/Documents/MS_calico/X0136/X0136-M004A-GPX4-lipids/X0136-M004A-2_blank.mzML"
MassCalcWidget::compute()
Found Blank: /Users/ngoc/Documents/MS_calico/X0136/X0136-M004A-GPX4-lipids/X0136-M004A-2_blank.mzML
loadSample time (msec) =  511
enumerateSRMScans: 11230
enumerateSRMScans: 11230
setIonizationMode:  -1
Loading sample: "/Users/ngoc/Documents/MS_calico/X0136/X0136-M004A-GPX4-lipids/X0136-M004A-3_st008A.mzML"
loadSample time (msec) =  523
enumerateSRMScans: 12511
enumerateSRMScans: 12511
setIonizationMode:  -1
Loading sample: "/Users/ngoc/Documents/MS_calico/X0136/X0136-M004A-GPX4-lipids/X0136-M004A-4_34-media.mzML"
loadSample time (msec) =  519
enumerateSRMScans: 11776
enumerateSRMScans: 11776
setIonizationMode:  -1
Loading sample: "/Users/ngoc/Documents/MS_calico/X0136/X0136-M004A-GPX4-lipids/X0136-M004A-5_35-plate.mzML"
loadSample time (msec) =  618
enumerateSRMScans: 11713
enumerateSRMScans: 11713
setIonizationMode:  -1
Loading sample: "/Users/ngoc/Documents/MS_calico/X0136/X0136-M004A-GPX4-lipids/X0136-M004A-6_25-AGPAT2-4.mzML"
loadSample time (msec) =  566
enumerateSRMScans: 12542
enumerateSRMScans: 12542
setIonizationMode:  -1
Loading sample: "/Users/ngoc/Documents/MS_calico/X0136/X0136-M004A-GPX4-lipids/X0136-M004A-7_7-ACSL4-4.mzML"
loadSample time (msec) =  450
enumerateSRMScans: 12615
enumerateSRMScans: 12615
setIonizationMode:  -1
Loading sample: "/Users/ngoc/Documents/MS_calico/X0136/X0136-M004A-GPX4-lipids/X0136-M004A-8_30-AGPAT3-1.mzML"
loadSample time (msec) =  553
enumerateSRMScans: 12671
enumerateSRMScans: 12671
setIonizationMode:  -1
Loading sample: "/Users/ngoc/Documents/MS_calico/X0136/X0136-M004A-GPX4-lipids/X0136-M004A-8a_load.mzML"
loadSample time (msec) =  470
enumerateSRMScans: 11010
enumerateSRMScans: 11010
setIonizationMode:  -1
Loading sample: "/Users/ngoc/Documents/MS_calico/X0136/X0136-M004A-GPX4-lipids/X0136-M004A-9_11-ACSL3-4.mzML"
loadSample time (msec) =  501
enumerateSRMScans: 12505
enumerateSRMScans: 12505
setIonizationMode:  -1
Loading sample: "/Users/ngoc/Documents/MS_calico/X0136/X0136-M004A-GPX4-lipids/X0136-M004A-10_4-ACSL4-3.mzML"
loadSample time (msec) =  451
enumerateSRMScans: 12569
enumerateSRMScans: 12569
setIonizationMode:  -1
Loading sample: "/Users/ngoc/Documents/MS_calico/X0136/X0136-M004A-GPX4-lipids/X0136-M004A-11_1-CD81-2.mzML"
loadSample time (msec) =  503
enumerateSRMScans: 12385
enumerateSRMScans: 12385
setIonizationMode:  -1
Loading sample: "/Users/ngoc/Documents/MS_calico/X0136/X0136-M004A-GPX4-lipids/X0136-M004A-12_29-AGPAT3-1.mzML"
loadSample time (msec) =  451
enumerateSRMScans: 12501
enumerateSRMScans: 12501
setIonizationMode:  -1
Loading sample: "/Users/ngoc/Documents/MS_calico/X0136/X0136-M004A-GPX4-lipids/X0136-M004A-13_18-AGPAT1-2.mzML"
loadSample time (msec) =  456
enumerateSRMScans: 12618
enumerateSRMScans: 12618
setIonizationMode:  -1
Loading sample: "/Users/ngoc/Documents/MS_calico/X0136/X0136-M004A-GPX4-lipids/X0136-M004A-14_load.mzML"
loadSample time (msec) =  505
enumerateSRMScans: 11450
enumerateSRMScans: 11450
setIonizationMode:  -1
Loading sample: "/Users/ngoc/Documents/MS_calico/X0136/X0136-M004A-GPX4-lipids/X0136-M004A-15_st008A.mzML"
loadSample time (msec) =  458
enumerateSRMScans: 12592
enumerateSRMScans: 12592
setIonizationMode:  -1
Loading sample: "/Users/ngoc/Documents/MS_calico/X0136/X0136-M004A-GPX4-lipids/X0136-M004A-16_26-AGPAT2-4.mzML"
loadSample time (msec) =  460
enumerateSRMScans: 12568
enumerateSRMScans: 12568
setIonizationMode:  -1
Loading sample: "/Users/ngoc/Documents/MS_calico/X0136/X0136-M004A-GPX4-lipids/X0136-M004A-17_6-ACSL4-3.mzML"
loadSample time (msec) =  458
enumerateSRMScans: 12464
enumerateSRMScans: 12464
setIonizationMode:  -1
Loading sample: "/Users/ngoc/Documents/MS_calico/X0136/X0136-M004A-GPX4-lipids/X0136-M004A-18_16-AGPAT1-2.mzML"
loadSample time (msec) =  514
enumerateSRMScans: 12606
enumerateSRMScans: 12606
setIonizationMode:  -1
Loading sample: "/Users/ngoc/Documents/MS_calico/X0136/X0136-M004A-GPX4-lipids/X0136-M004A-19_21-AGPAT1-4.mzML"
loadSample time (msec) =  445
enumerateSRMScans: 12654
enumerateSRMScans: 12654
setIonizationMode:  -1
Loading sample: "/Users/ngoc/Documents/MS_calico/X0136/X0136-M004A-GPX4-lipids/X0136-M004A-21_31-AGPAT3-5.mzML"
loadSample time (msec) =  560
enumerateSRMScans: 12617
enumerateSRMScans: 12617
setIonizationMode:  -1
Loading sample: "/Users/ngoc/Documents/MS_calico/X0136/X0136-M004A-GPX4-lipids/X0136-M004A-22_13-ACSL3-5.mzML"
loadSample time (msec) =  450
enumerateSRMScans: 12485
enumerateSRMScans: 12485
setIonizationMode:  -1
Loading sample: "/Users/ngoc/Documents/MS_calico/X0136/X0136-M004A-GPX4-lipids/X0136-M004A-23_15-ACSL3-5.mzML"
loadSample time (msec) =  580
enumerateSRMScans: 12500
enumerateSRMScans: 12500
setIonizationMode:  -1
Loading sample: "/Users/ngoc/Documents/MS_calico/X0136/X0136-M004A-GPX4-lipids/X0136-M004A-24_33-AGPAT3-5.mzML"
loadSample time (msec) =  452
enumerateSRMScans: 12666
enumerateSRMScans: 12666
setIonizationMode:  -1
Loading sample: "/Users/ngoc/Documents/MS_calico/X0136/X0136-M004A-GPX4-lipids/X0136-M004A-25_23-AGPAT2-1.mzML"
loadSample time (msec) =  480
enumerateSRMScans: 12513
enumerateSRMScans: 12513
setIonizationMode:  -1
Loading sample: "/Users/ngoc/Documents/MS_calico/X0136/X0136-M004A-GPX4-lipids/X0136-M004A-26_10-ACSL3-4.mzML"
loadSample time (msec) =  472
enumerateSRMScans: 12538
enumerateSRMScans: 12538
setIonizationMode:  -1
Loading sample: "/Users/ngoc/Documents/MS_calico/X0136/X0136-M004A-GPX4-lipids/X0136-M004A-27_load.mzML"
loadSample time (msec) =  499
enumerateSRMScans: 11173
enumerateSRMScans: 11173
setIonizationMode:  -1
Loading sample: "/Users/ngoc/Documents/MS_calico/X0136/X0136-M004A-GPX4-lipids/X0136-M004A-28_st008A.mzML"
loadSample time (msec) =  473
enumerateSRMScans: 12571
enumerateSRMScans: 12571
setIonizationMode:  -1
Loading sample: "/Users/ngoc/Documents/MS_calico/X0136/X0136-M004A-GPX4-lipids/X0136-M004A-29_14-ACSL3-5.mzML"
loadSample time (msec) =  467
enumerateSRMScans: 12466
enumerateSRMScans: 12466
setIonizationMode:  -1
Loading sample: "/Users/ngoc/Documents/MS_calico/X0136/X0136-M004A-GPX4-lipids/X0136-M004A-30_12-ACSL3-4.mzML"
loadSample time (msec) =  632
enumerateSRMScans: 12409
enumerateSRMScans: 12409
setIonizationMode:  -1
Loading sample: "/Users/ngoc/Documents/MS_calico/X0136/X0136-M004A-GPX4-lipids/X0136-M004A-31_3-CD81-2.mzML"
loadSample time (msec) =  558
enumerateSRMScans: 12224
enumerateSRMScans: 12224
setIonizationMode:  -1
Loading sample: "/Users/ngoc/Documents/MS_calico/X0136/X0136-M004A-GPX4-lipids/X0136-M004A-32_24-AGPAT2-1.mzML"
loadSample time (msec) =  497
enumerateSRMScans: 12411
enumerateSRMScans: 12411
setIonizationMode:  -1
Loading sample: "/Users/ngoc/Documents/MS_calico/X0136/X0136-M004A-GPX4-lipids/X0136-M004A-33_28-AGPAT3-1.mzML"
loadSample time (msec) =  547
enumerateSRMScans: 12533
enumerateSRMScans: 12533
setIonizationMode:  -1
Loading sample: "/Users/ngoc/Documents/MS_calico/X0136/X0136-M004A-GPX4-lipids/X0136-M004A-34_20-AGPAT1-4.mzML"
loadSample time (msec) =  486
enumerateSRMScans: 12525
enumerateSRMScans: 12525
setIonizationMode:  -1
Loading sample: "/Users/ngoc/Documents/MS_calico/X0136/X0136-M004A-GPX4-lipids/X0136-M004A-35_8-ACSL4-4.mzML"
loadSample time (msec) =  587
enumerateSRMScans: 12475
enumerateSRMScans: 12475
setIonizationMode:  -1
Loading sample: "/Users/ngoc/Documents/MS_calico/X0136/X0136-M004A-GPX4-lipids/X0136-M004A-36_27-AGPAT2-4.mzML"
loadSample time (msec) =  553
enumerateSRMScans: 12526
enumerateSRMScans: 12526
setIonizationMode:  -1
Loading sample: "/Users/ngoc/Documents/MS_calico/X0136/X0136-M004A-GPX4-lipids/X0136-M004A-37_2-CD81-2.mzML"
loadSample time (msec) =  541
enumerateSRMScans: 12368
enumerateSRMScans: 12368
setIonizationMode:  -1
Loading sample: "/Users/ngoc/Documents/MS_calico/X0136/X0136-M004A-GPX4-lipids/X0136-M004A-38_17-AGPAT1-2.mzML"
loadSample time (msec) =  688
enumerateSRMScans: 12455
enumerateSRMScans: 12455
setIonizationMode:  -1
Loading sample: "/Users/ngoc/Documents/MS_calico/X0136/X0136-M004A-GPX4-lipids/X0136-M004A-39_19-AGPAT1-4.mzML"
loadSample time (msec) =  518
enumerateSRMScans: 12474
enumerateSRMScans: 12474
setIonizationMode:  -1
Loading sample: "/Users/ngoc/Documents/MS_calico/X0136/X0136-M004A-GPX4-lipids/X0136-M004A-40_5-ACSL4-3.mzML"
loadSample time (msec) =  572
enumerateSRMScans: 12391
enumerateSRMScans: 12391
setIonizationMode:  -1
Loading sample: "/Users/ngoc/Documents/MS_calico/X0136/X0136-M004A-GPX4-lipids/X0136-M004A-41_32-AGPAT3-5.mzML"
loadSample time (msec) =  502
enumerateSRMScans: 12553
enumerateSRMScans: 12553
setIonizationMode:  -1
Loading sample: "/Users/ngoc/Documents/MS_calico/X0136/X0136-M004A-GPX4-lipids/X0136-M004A-42_9-ACSL4-4.mzML"
loadSample time (msec) =  467
enumerateSRMScans: 12543
enumerateSRMScans: 12543
setIonizationMode:  -1
Loading sample: "/Users/ngoc/Documents/MS_calico/X0136/X0136-M004A-GPX4-lipids/X0136-M004A-43_load.mzML"
loadSample time (msec) =  489
enumerateSRMScans: 11090
enumerateSRMScans: 11090
setIonizationMode:  -1
Loading sample: "/Users/ngoc/Documents/MS_calico/X0136/X0136-M004A-GPX4-lipids/X0136-M004A-44_st008A.mzML"
loadSample time (msec) =  495
enumerateSRMScans: 12383
enumerateSRMScans: 12383
setIonizationMode:  -1
Loading sample: "/Users/ngoc/Documents/MS_calico/X0136/X0136-M004A-GPX4-lipids/X0136-M004A-45_blank.mzML"
Found Blank: /Users/ngoc/Documents/MS_calico/X0136/X0136-M004A-GPX4-lipids/X0136-M004A-45_blank.mzML
loadSample time (msec) =  452
enumerateSRMScans: 11049
enumerateSRMScans: 11049
setIonizationMode:  -1
ProjectDockWidget::warnUserEmptySampleFiles()
getSampleInfoSQLITE() 0
Loading sampleId 0 "X0136-M004A-1_blank.mzML"
Loading sampleId 1 "X0136-M004A-2_blank.mzML"
Loading sampleId 2 "X0136-M004A-3_st008A.mzML"
Loading sampleId 3 "X0136-M004A-4_34-media.mzML"
Loading sampleId 4 "X0136-M004A-5_35-plate.mzML"
Loading sampleId 5 "X0136-M004A-6_25-AGPAT2-4.mzML"
Loading sampleId 6 "X0136-M004A-7_7-ACSL4-4.mzML"
Loading sampleId 7 "X0136-M004A-8_30-AGPAT3-1.mzML"
Loading sampleId 8 "X0136-M004A-8a_load.mzML"
Loading sampleId 9 "X0136-M004A-9_11-ACSL3-4.mzML"
Loading sampleId 10 "X0136-M004A-10_4-ACSL4-3.mzML"
Loading sampleId 11 "X0136-M004A-11_1-CD81-2.mzML"
Loading sampleId 12 "X0136-M004A-12_29-AGPAT3-1.mzML"
Loading sampleId 13 "X0136-M004A-13_18-AGPAT1-2.mzML"
Loading sampleId 14 "X0136-M004A-14_load.mzML"
Loading sampleId 15 "X0136-M004A-15_st008A.mzML"
Loading sampleId 16 "X0136-M004A-16_26-AGPAT2-4.mzML"
Loading sampleId 17 "X0136-M004A-17_6-ACSL4-3.mzML"
Loading sampleId 18 "X0136-M004A-18_16-AGPAT1-2.mzML"
Loading sampleId 19 "X0136-M004A-19_21-AGPAT1-4.mzML"
Loading sampleId 20 "X0136-M004A-21_31-AGPAT3-5.mzML"
Loading sampleId 21 "X0136-M004A-22_13-ACSL3-5.mzML"
Loading sampleId 22 "X0136-M004A-23_15-ACSL3-5.mzML"
Loading sampleId 23 "X0136-M004A-24_33-AGPAT3-5.mzML"
Loading sampleId 24 "X0136-M004A-25_23-AGPAT2-1.mzML"
Loading sampleId 25 "X0136-M004A-26_10-ACSL3-4.mzML"
Loading sampleId 26 "X0136-M004A-27_load.mzML"
Loading sampleId 27 "X0136-M004A-28_st008A.mzML"
Loading sampleId 28 "X0136-M004A-29_14-ACSL3-5.mzML"
Loading sampleId 29 "X0136-M004A-30_12-ACSL3-4.mzML"
Loading sampleId 30 "X0136-M004A-31_3-CD81-2.mzML"
Loading sampleId 31 "X0136-M004A-32_24-AGPAT2-1.mzML"
Loading sampleId 32 "X0136-M004A-33_28-AGPAT3-1.mzML"
Loading sampleId 33 "X0136-M004A-34_20-AGPAT1-4.mzML"
Loading sampleId 34 "X0136-M004A-35_8-ACSL4-4.mzML"
Loading sampleId 35 "X0136-M004A-36_27-AGPAT2-4.mzML"
Loading sampleId 36 "X0136-M004A-37_2-CD81-2.mzML"
Loading sampleId 37 "X0136-M004A-38_17-AGPAT1-2.mzML"
Loading sampleId 38 "X0136-M004A-39_19-AGPAT1-4.mzML"
Loading sampleId 39 "X0136-M004A-40_5-ACSL4-3.mzML"
Loading sampleId 40 "X0136-M004A-41_32-AGPAT3-5.mzML"
Loading sampleId 41 "X0136-M004A-42_9-ACSL4-4.mzML"
Loading sampleId 42 "X0136-M004A-43_load.mzML"
Loading sampleId 43 "X0136-M004A-44_st008A.mzML"
Loading sampleId 44 "X0136-M004A-45_blank.mzML"
ProjectDB::doAlignment() loaded 2829 segments	Aligner::doSegmentedAligment()samples=45
Can't find segment for: X0136-M004A-1_blank.mzML	60.0085
doSegmentedAligment: X0136-M004A-1_blank.mzML	corrected=11251
Can't find segment for: X0136-M004A-2_blank.mzML	60.0006
doSegmentedAligment: X0136-M004A-2_blank.mzML	corrected=11229
Can't find segment for: X0136-M004A-3_st008A.mzML	60.0158
doSegmentedAligment: X0136-M004A-3_st008A.mzML	corrected=12510
Can't find segment for: X0136-M004A-4_34-media.mzML	60.0085
doSegmentedAligment: X0136-M004A-4_34-media.mzML	corrected=11775
Can't find segment for: X0136-M004A-5_35-plate.mzML	60.0028
doSegmentedAligment: X0136-M004A-5_35-plate.mzML	corrected=11712
Can't find segment for: X0136-M004A-6_25-AGPAT2-4.mzML	60.0101
doSegmentedAligment: X0136-M004A-6_25-AGPAT2-4.mzML	corrected=12541
Can't find segment for: X0136-M004A-7_7-ACSL4-4.mzML	60.008
doSegmentedAligment: X0136-M004A-7_7-ACSL4-4.mzML	corrected=12614
Can't find segment for: X0136-M004A-8_30-AGPAT3-1.mzML	60.0101
doSegmentedAligment: X0136-M004A-8_30-AGPAT3-1.mzML	corrected=12670
Can't find segment for: X0136-M004A-8a_load.mzML	60.0093
doSegmentedAligment: X0136-M004A-8a_load.mzML	corrected=11009
Can't find segment for: X0136-M004A-9_11-ACSL3-4.mzML	60.0078
doSegmentedAligment: X0136-M004A-9_11-ACSL3-4.mzML	corrected=12504
Can't find segment for: X0136-M004A-10_4-ACSL4-3.mzML	60.0008
doSegmentedAligment: X0136-M004A-10_4-ACSL4-3.mzML	corrected=12568
Can't find segment for: X0136-M004A-11_1-CD81-2.mzML	60.0065
doSegmentedAligment: X0136-M004A-11_1-CD81-2.mzML	corrected=12384
Can't find segment for: X0136-M004A-12_29-AGPAT3-1.mzML	60.0067
doSegmentedAligment: X0136-M004A-12_29-AGPAT3-1.mzML	corrected=12500
Can't find segment for: X0136-M004A-13_18-AGPAT1-2.mzML	60.0002
doSegmentedAligment: X0136-M004A-13_18-AGPAT1-2.mzML	corrected=12617
Can't find segment for: X0136-M004A-14_load.mzML	60.004
doSegmentedAligment: X0136-M004A-14_load.mzML	corrected=11449
Can't find segment for: X0136-M004A-15_st008A.mzML	60.0097
doSegmentedAligment: X0136-M004A-15_st008A.mzML	corrected=12591
Can't find segment for: X0136-M004A-16_26-AGPAT2-4.mzML	60.0084
doSegmentedAligment: X0136-M004A-16_26-AGPAT2-4.mzML	corrected=12567
Can't find segment for: X0136-M004A-17_6-ACSL4-3.mzML	60.0016
doSegmentedAligment: X0136-M004A-17_6-ACSL4-3.mzML	corrected=12463
Can't find segment for: X0136-M004A-18_16-AGPAT1-2.mzML	60.0079
doSegmentedAligment: X0136-M004A-18_16-AGPAT1-2.mzML	corrected=12605
Can't find segment for: X0136-M004A-19_21-AGPAT1-4.mzML	60.01
doSegmentedAligment: X0136-M004A-19_21-AGPAT1-4.mzML	corrected=12653
Can't find segment for: X0136-M004A-21_31-AGPAT3-5.mzML	60.001
doSegmentedAligment: X0136-M004A-21_31-AGPAT3-5.mzML	corrected=12616
Can't find segment for: X0136-M004A-22_13-ACSL3-5.mzML	59.9996
doSegmentedAligment: X0136-M004A-22_13-ACSL3-5.mzML	corrected=12484
Can't find segment for: X0136-M004A-23_15-ACSL3-5.mzML	60.0121
doSegmentedAligment: X0136-M004A-23_15-ACSL3-5.mzML	corrected=12499
Can't find segment for: X0136-M004A-24_33-AGPAT3-5.mzML	60.0004
doSegmentedAligment: X0136-M004A-24_33-AGPAT3-5.mzML	corrected=12665
Can't find segment for: X0136-M004A-25_23-AGPAT2-1.mzML	59.9953
doSegmentedAligment: X0136-M004A-25_23-AGPAT2-1.mzML	corrected=12512
Can't find segment for: X0136-M004A-26_10-ACSL3-4.mzML	60.002
doSegmentedAligment: X0136-M004A-26_10-ACSL3-4.mzML	corrected=12537
Can't find segment for: X0136-M004A-27_load.mzML	60.0026
doSegmentedAligment: X0136-M004A-27_load.mzML	corrected=11172
Can't find segment for: X0136-M004A-28_st008A.mzML	60
doSegmentedAligment: X0136-M004A-28_st008A.mzML	corrected=12570
Can't find segment for: X0136-M004A-29_14-ACSL3-5.mzML	60.0099
doSegmentedAligment: X0136-M004A-29_14-ACSL3-5.mzML	corrected=12465
Can't find segment for: X0136-M004A-30_12-ACSL3-4.mzML	59.9977
doSegmentedAligment: X0136-M004A-30_12-ACSL3-4.mzML	corrected=12408
Can't find segment for: X0136-M004A-31_3-CD81-2.mzML	60.0053
doSegmentedAligment: X0136-M004A-31_3-CD81-2.mzML	corrected=12223
Can't find segment for: X0136-M004A-32_24-AGPAT2-1.mzML	60.0136
doSegmentedAligment: X0136-M004A-32_24-AGPAT2-1.mzML	corrected=12410
Can't find segment for: X0136-M004A-33_28-AGPAT3-1.mzML	60.0032
doSegmentedAligment: X0136-M004A-33_28-AGPAT3-1.mzML	corrected=12532
Can't find segment for: X0136-M004A-34_20-AGPAT1-4.mzML	60.0055
doSegmentedAligment: X0136-M004A-34_20-AGPAT1-4.mzML	corrected=12524
Can't find segment for: X0136-M004A-35_8-ACSL4-4.mzML	60.0013
doSegmentedAligment: X0136-M004A-35_8-ACSL4-4.mzML	corrected=12474
Can't find segment for: X0136-M004A-36_27-AGPAT2-4.mzML	59.9922
doSegmentedAligment: X0136-M004A-36_27-AGPAT2-4.mzML	corrected=12525
Can't find segment for: X0136-M004A-37_2-CD81-2.mzML	60.0024
doSegmentedAligment: X0136-M004A-37_2-CD81-2.mzML	corrected=12367
Can't find segment for: X0136-M004A-38_17-AGPAT1-2.mzML	60.0003
doSegmentedAligment: X0136-M004A-38_17-AGPAT1-2.mzML	corrected=12454
Can't find segment for: X0136-M004A-39_19-AGPAT1-4.mzML	60.0111
doSegmentedAligment: X0136-M004A-39_19-AGPAT1-4.mzML	corrected=12473
Can't find segment for: X0136-M004A-40_5-ACSL4-3.mzML	60.0176
doSegmentedAligment: X0136-M004A-40_5-ACSL4-3.mzML	corrected=12390
Can't find segment for: X0136-M004A-41_32-AGPAT3-5.mzML	60.0071
doSegmentedAligment: X0136-M004A-41_32-AGPAT3-5.mzML	corrected=12552
Can't find segment for: X0136-M004A-42_9-ACSL4-4.mzML	60.0184
doSegmentedAligment: X0136-M004A-42_9-ACSL4-4.mzML	corrected=12542
Can't find segment for: X0136-M004A-43_load.mzML	60.0098
doSegmentedAligment: X0136-M004A-43_load.mzML	corrected=11089
Can't find segment for: X0136-M004A-44_st008A.mzML	60.0032
doSegmentedAligment: X0136-M004A-44_st008A.mzML	corrected=12382
Can't find segment for: X0136-M004A-45_blank.mzML	60.0076
doSegmentedAligment: X0136-M004A-45_blank.mzML	corrected=11048
Search Table Names:
entry= "Bookmarks"
entry= "peakdetector"
entry= "rumsDB"
End Search Table Names.
Created tables for search results.
Cleared loaded peak groups.
ProjectDB::loadPeakGroups(): isHasDisplayName?  true
Database::loadCompoundsSQL(): Elapsed time after count query: 1 msec
Database::loadCompoundSQL() Finished reading in data for "X0136-M004A-GPX4-lipids_all_compounds"
Database::loadCompoundSQL(): Finished appending data for "X0136-M004A-GPX4-lipids_all_compounds" to compoundDB in 1 msec. size= 1767
Database::loadCompoundsSQL(): Elapsed time after count query: 0 msec
Database::loadCompoundSQL() Finished reading in data for "X0136-M004A-GPX4-lipids_all_compounds"
Database::loadCompoundSQL(): Finished appending data for "X0136-M004A-GPX4-lipids_all_compounds" to compoundDB in 0 msec. size= 1767
Database::loadCompoundsSQL(): Elapsed time after count query: 0 msec
Database::loadCompoundSQL() Finished reading in data for "X0136-M004A-GPX4-lipids_all_compounds"
Database::loadCompoundSQL(): Finished appending data for "X0136-M004A-GPX4-lipids_all_compounds" to compoundDB in 0 msec. size= 1767
Database::loadCompoundsSQL(): Elapsed time after count query: 0 msec
Database::loadCompoundSQL() Finished reading in data for "X0136-M004A-GPX4-lipids_all_compounds"
Database::loadCompoundSQL(): Finished appending data for "X0136-M004A-GPX4-lipids_all_compounds" to compoundDB in 0 msec. size= 1767
Database::loadCompoundsSQL(): Elapsed time after count query: 3192 msec
Database::loadCompoundSQL() Finished reading in data for "20191022-Calico-Lipids-neg"
Segmentation fault: 11
logout
Saving session...
...copying shared history...
...saving history...truncating history files...
...completed.
````

**Comment by PMSeitzer** [comment]
The bug is happening during sorting of the compound DB. probably a `nullptr` access, but I'm confused as to how this could happen.

**Comment by ngoccalico** [comment]
Hi @PMSeitzer , it is true I cannot open any library. everytime I tried to open the library Maven Crashed

**Comment by PMSeitzer** [comment]
Fixed as of 45a76395a8e315846f3e3a22a3d60aefc4296f03
